bin/check-ntp.rb

```diff
@@ -45,9 +50,15 @@ def run
     rescue
       unknown 'NTP command Failed'
     end
-
-    critical 'NTP not synced' if stratum > 15
-
+  
+    if stratum > 15
+      critical "NTP not synced"
+    elsif config[:stratum]
+      if stratum > config[:stratum]
+        critical "NTP stratum (#{stratum}) above limit (#{config[:stratum]})"
+      end
+    end
```

Comment:
Any reason not to just make the stratum config have a default value of 15.  It seems like this would avoid some complexity here.